pkg/controller.v1/tensorflow/pod.go

```diff
@@ -226,19 +231,47 @@ func setClusterSpec(podTemplateSpec *v1.PodTemplateSpec, tfjob *tfv1.TFJob, rt,
 	if tfConfigStr == "" {
 		return nil
 	}
-	// Add TF_CONFIG environment variable.
+	// Add TF_CONFIG environment variable to tensorflow container in the pod.
```

Comment:
This patch might be needed in Pytorch also.

Reply:
I remember that you said it does not have a side effect in PyTorch. Do we need it?